bedrock/foundation/templates/foundation/trademarks/policy.html

```diff
@@ -76,34 +74,33 @@ <h4>{{ _('Don’t:') }}</h4>
+    <li>{{ _('Don’t use marks, logos, company names, slogans, domain names, or designs that are confusingly similar to Mozilla trademarks.') }}</li>
 
-    <li>{{ _('Don’t display Mozilla trademarks more prominently than your product or service name.') }}</li>
+    <li>{{ _('Don’t use Mozilla trademarks in a way that incorrectly implies affiliation with, or sponsorship, endorsement, or approval by Mozilla of your products or services.') }}</li>
 
-    <li>{{ _('Don’t use Mozilla trademarks on merchandise for sale (e.g., selling t-shirts, mugs, etc.)
-or other commercial use.') }}</li>
+    <li>{{ _('Don’t display Mozilla trademarks more prominently than your product, service, or company name.') }}</li>
 
-    <li>{{ _('Don’t modify Mozilla’s trademarks, combine them with any other symbols, words, or images, or incorporate them into a tagline or slogan.') }}</li>
+    <li>{{ _('Don’t use Mozilla trademarks on merchandise for sale (e.g., selling t-shirts, mugs, etc.)') }}</li>
 
-    <li>{{ _('Don’t use Mozilla trademarks in a way that is misleading, unfair, defamatory, libelous, disparaging, obscene, or sullies Mozilla’s reputation.') }}</li>
+    <li>{{ _('Don’t use Mozilla trademarks for any other form of commercial use (e.g. offering technical support services), unless such use is limited to a truthful and descriptive reference (e.g. “Independent technical support for Mozilla’s Firefox browser”).') }}</li>
 
-    <li>{{ _('Don’t use Mozilla trademarks in association with any illegal activities or materials.') }}</li>
+    <li>{{ _('Don’t modify Mozilla’s trademarks, abbreviate them, or combine them with any other symbols, words, or images, or incorporate them into a tagline or slogan.') }}</li>
 
   </ul>
 
   <h3>{{ _('Additional Guidelines') }}</h3>
 
+  <h4>{{ _('Software Distribution Guidelines') }}</h4>
+
+  <p>{% trans distribution=url('foundation.trademarks.distribution-policy') %}<a href="{{ distribution }}">Click here</a> for additional guidelines that apply to distribution of the Firefox Browser and other Mozilla software.{% endtrans %} </p>
+
   <h4>{{ _('Mozilla Logo and Firefox Logo Guidelines') }}</h4>
 
-  <p>{{ _('In addition to the General Guidelines above, the Mozilla Style Guide has detailed information on proper usage of the Mozilla logo and the Firefox logo. See the respective pages for more information:') }}</p>
+  <p>{{ _('If your use of a logo is permitted under the General Guidelines above, hen please refer to the Mozilla Style Guide for detailed information on proper usage.') }}</p>
 
-  <p>{% trans mlogo=("https://mozilla.design/mlogo") %}Mozilla Logo: <a href="{{ mlogo }}">https://mozilla.design/mlogo</a>{% endtrans %}</p>
+  <p>{% trans mlogo=("https://mozilla.design/mozilla/logo-usage/") %}Mozilla Logo: <a href="{{ mlogo }}">https://mozilla.design/mozilla/logo-usage/</a>{% endtrans %}</p>
```

Comment:
I think we can probably exclude the link from the translated string here? (also applies to the firefox logo link below).

```suggestion
  <p>{{ _('Mozilla Logo:') }} <a href="https://mozilla.design/mozilla/logo-usage/">https://mozilla.design/mozilla/logo-usage/</a></p>
```